meetings/2025-03-05.md

```diff
@@ -0,0 +1,121 @@
+* JW: For N3 patch this isn't much of an issue. We don't propose that deletions are allowed to refer to blank node. Besides the patch identifier itself there are no blank nodes in the document. We can propose a compromise that bypasses all of the issue mentioned by Ruben.
+* JW: Dorthe have commented that there is an interest in N3 WG that would resolve that issue with blank node scoping. We can procede with the insertion and be aware with the pitfalls in current parsers.
+* MdJ: Have anyone implemented it yet?
+* JW: NSS always allowed it, CSS forbids it. CSS uses N3 which can parse those blank nodes, there is just one line in CSS that checks if the blank nodes exist. I understand Ruben's argument but I see it as academic precission.
+* MdJ: If NSS already does it, if CSS doesn't do it we can do it in Pivot. ESS doesn't do N3 patch at all.
+* eP: Do you know why ESS doesn't support it?
+* MdJ: The whole patching is in the air. When the switch away from SPARQL update happended, it was known that WG will be resolving that. So this wasn't a priority to implement what was proposed in CG.
+* RG: Why is it that we put N3 patch as specification inside solid protocol spec instead having it as a normative reference. Jesse can you make that comment about planned change in the PR as a note?
+* MdJ: Why don't we separate it from the protocol spec. It seems to match suggestion that there will be a N3 WG.
+* RG: Would they also take responsibility of N3 patch?
+* MdJ: That would also help to keep Solid CG on topic. If there is an N3 CG we could move the patch there. In the protocol we can make a chance to refer to it as a normative reference.
+* JW: I don't think N3 CG would be interested in patch to be honset. It is describing how to modify resources. There is no other write type interface in N3 spec at this moment. It defines semantic, inference and query. It doesn't describe any mechanism to update.
+* RG: For now we just make a change within the Solid Spec, we take it in Solid CG and take it into a separate document. Once we have N3 Patch document we can ask them if they want to do it. Or we can hand it to LWS.
+* eP: +1 to extract it as a separate draft report
+* JW: That sounds reasonable
+* MdJ: can we make that change in Pivot
+* JW: I don't have time at this moment. I can look into it at some poiont. 
+* MdJ: Since NSS already does it, we should also do it in Pivot
+* HZ: Sounds good.
+* MdJ: We are just incubating here as CG ...
+* eP: Are we going to merge that PR?
+* RG: I can do it but not before April 1st or 3rd
+* eP: So we merge the PR first and later we extract it?
+* RG: Yes, merge PR first and do extraction later.
+* RG: I can do the mechanical extraction but I can't be the editor since I'm still new to RDF
+* JW: I can help with RDF bits
```

Comment:
```suggestion
* RG: I can do it but not before April 1st or 3rd.
* eP: So we merge the PR first and later we extract it?
* RG: Yes, merge PR first and do extraction later.
* RG: I can do the mechanical extraction, but I can't be the editor, since I'm still new to RDF.
* JW: I can help with RDF bits.
```